### PR TITLE
Shield rework

### PR DESCRIPTION
**Pull request by Uberbagel**
This is the first phase of shield damage and shield condition rework.

Phase One: Rebalance  Shield Damage and Shield Condition to make both shields and shield splitting useful.

Phase Two: Increase Shield Damage on regular attacks to be both useful and balanced.

Phase Three: Shield Damage that exceeds a Shield's remaining condition deals the remaining damage to the target.

This PR:
I've balanced our current lineup against each other and against each shield in the game. I feel strongly I have a pretty good balance here. You can see the whole list of shield damage by weapon in the weapons spreadsheet. You can use this link to quickly see shield conditions (https://battlebrothers.fandom.com/wiki/Armor#Shields). If the numbers are too hard to parse and come on up with an understanding of the balance, let me know and I can expand upon my design intentions.

Edit: I forgot I also rebalanced orc axe fatigue cost because it was wrong. I can split up this commit if necessary.

### CODE REVIEW

**Comment by Darxo** [comment]
These numbers are hard to parse for me.

I don't know what problems of the vanilla values you are trying to fix
You state the goal of you wanting

> to make both shields and shield splitting useful

So, how useful do you see shields before and after your change? And likewise with Shield Splitting.

Also for those balance passes some summary would be very helpful for the other devs. Something like these types of items/skills are on average this % weaker/stronger.

**Comment by LordMidas** [comment]
We did an overview together and added the changes discussed in our meeting.
